docker/test/monolith.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+rm -rf build
```

Comment:
Is this clearing the directory `/path/to/dendrite/build` when the container is started through `docker-compose-test.yml` above? I don't see that directory is used somewhere so this is a bit confusing to me.